apps/hyperdrive-trading/src/ui/token/TokenPickerTwo.tsx

```diff
@@ -111,3 +116,114 @@ export function TokenPickerTwo({
     </div>
   );
 }
+
+export function ZapsTokenPicker({
+  tokens,
+  activeTokenAddress,
+  onChange,
+  label,
+}: {
+  tokens: TokenChoice[];
+  activeTokenAddress: Address;
+  onChange: (tokenAddress: Address) => void;
+  label?: string;
+}): ReactElement {
+  const [searchString, setSearchString] = useState("");
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  const fuse = useMemo(() => {
+    return new Fuse(tokens, {
+      keys: ["tokenConfig.symbol", "tokenConfig.name"],
+      threshold: 0.4,
+    });
+  }, [tokens]);
+
+  const filteredTokens = searchString
+    ? fuse.search(searchString).map(({ item }) => item)
+    : tokens;
+
+  const activeToken = tokens.find(
+    ({ tokenConfig }) => tokenConfig?.address === activeTokenAddress,
+  );
+
+  return (
+    <div>
+      {label ? (
+        <label className="daisy-label">
+          <span className="daisy-label-text">{label}</span>
+        </label>
+      ) : undefined}
+      <div className="daisy-dropdown daisy-dropdown-end daisy-dropdown-bottom">
+        <button
+          className={classNames(
+            "daisy-btn daisy-btn-md flex h-9 min-h-9 items-center rounded-box bg-neutral pl-3 pr-1",
+          )}
+          onClick={(e) => {
+            e.preventDefault();
+          }}
```

Comment:
We should be able to remove this if the button's `type` is set to `button`:
```suggestion
          type="button"
```